tracker/storage_sqlite.go

```diff
@@ -0,0 +1,168 @@
+// +build sqlite
```

Comment:
I think it would be better to have a comment describing why we need a build tag for sqlite, especially if other teams use this library too